test-it/ExpectedResults/ComplexMathFunctions.vtr

```diff
@@ -40,3 +40,5 @@ Log (0 99) => (4.59512 1.5708)
 Log2 (0 99) => (6.62936 2.26618)
 Log10 (0 99) => (1.99564 0.682188)
 Log10 ((1.75517 0.958851) (1.62091 2.52441)) => ((0.30103 0.217147) (0.477121 0.434294))
+Absolute Double (-96.7324 -265.621) => 282.6865193998785912
```

Comment:
The following code seems to produce identical results on native system. Could you tell us what they are on the browser? I agree with spathiwa that we should keep the original code unless there is a significant difference.

#include <iostream>
#include <string>
#include <complex>
#include <cmath>
#include <limits>

using namespace std;

int main()
{
  complex<double> c1(-96.7324253138614551517 -265.620982121737029047);
  double abs1 = abs(c1);
  double abs2 = sqrt(c1.real()*c1.real() + c1.imag()*c1.imag());
  cout.precision(std::numeric_limits<double>::max_digits10);
  cout << fixed << abs1 << endl << abs2 << endl;
}